### DISCUSSION TITLE
What was the first James Bond movie in which Daniel Craig appeared?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#565] [Galaxy Trivia] [answered]
**Bonus:** What is the name of the last?

**Comment by peterbaccaro** [comment]
Daniel Craig first James Bond movie was **Casino Royale** in 2006.  His fifth and last James Bond movie was **No Time to Die** in 2021.

**Comment by thuongtruong1009** [comment, marked as answer]
* The first James Bond movie in which Daniel Craig appeared is "Casino Royale," which was released in 2006. It was the 21st film in the James Bond franchise and was directed by Martin Campbell. Craig's portrayal of the iconic character was well received, and he has since gone on to appear in four more James Bond movies, including "Quantum of Solace," "Skyfall," "Spectre," and "No Time to Die."

* The latest James Bond movie, released in 2021, is "No Time to Die." It is the 25th film in the James Bond franchise and marks Daniel Craig's fifth and final appearance as the character. The movie was directed by Cary Joji Fukunaga and received positive reviews from critics and audiences alike.

**Comment by Veercodeprog** [comment]
Casino royale
